samples/zigbee/ncp/CMakeLists.txt

```diff
@@ -6,6 +6,25 @@
 
 cmake_minimum_required(VERSION 3.13.1)
 
+if ("usb.overlay" IN_LIST OVERLAY_CONFIG OR ${BOARD} STREQUAL "nrf52840dongle_nrf52840")
+  if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static.yml")
+    set(PM_STATIC_YML_FILE
+      ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static.yml
+    )
+  endif()
+
+  if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/mcuboot.conf")
+    set(mcuboot_OVERLAY_CONFIG ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/mcuboot.conf)
+  endif()
+
+  if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/app_with_mcuboot.conf")
+    set(CONF_FILE
+      prj.conf
+      ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/app_with_mcuboot.conf
+    )
+  endif()
```

Comment:
Shall the mcuboot.conf and app_with_mcuboot.conf only be used for `usb.overlay` or the `nrf52840dongle_nrf52840` board ?

Or is this code on the wrong side of the `if ... endif` statement. 

Also, please adopt to this pattern for the child image:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html#image-specific-variables

and probably use a `prj_mcuboot.conf` for the main app, which allows for additional `<board>_mcuboot` Kconfig fragments to be automatically sourced:
https://docs.zephyrproject.org/latest/guides/kconfig/setting.html#the-initial-configuration

Feel free to ping me directly to discuss how to best meet your needs and keep a pattern similar to what is used elsewhere.